libs/pages/clusters/src/lib/ui/page-clusters-general/page-clusters-general.spec.tsx

```diff
@@ -0,0 +1,42 @@
+import { render } from '__tests__/utils/setup-jest'
+import { clusterFactoryMock } from '@qovery/domains/organization'
+import PageClustersGeneral, { PageClustersGeneralProps } from './page-clusters-general'
+
+describe('PageClustersGeneral', () => {
+  const props: PageClustersGeneralProps = {
+    clusters: clusterFactoryMock(2),
+    loading: 'loaded',
+  }
+
+  it('should render successfully', () => {
+    const { baseElement } = render(<PageClustersGeneral {...props} />)
+    expect(baseElement).toBeTruthy()
+  })
+
+  it('should have an loader spinner', () => {
+    props.loading = 'loading'
+    props.clusters = []
+
+    const { getByTestId } = render(<PageClustersGeneral {...props} />)
+
+    getByTestId('clusters-loader')
+  })
+
+  it('should have an empty screen', () => {
+    props.loading = 'loaded'
+    props.clusters = []
+
+    const { getByTestId } = render(<PageClustersGeneral {...props} />)
+
+    getByTestId('empty-state')
+  })
+
+  it('should have an list of registries', () => {
+    props.loading = 'loaded'
+    props.clusters = clusterFactoryMock(1)
```

Comment:
Here you had to recreate one cluster because it was empty from previous tests right?
You could have put the props setting in the beforeEach() which would have avoid you to do that here and that, in a more common case prevent some undesired weird behavior.
Here it's completely fine, just I wanted to give your this "tips" that lately I try to put the props in a beforeEach when I'm changing its values often like you do here.